ggd-recipes/GRCh37/gnomad.yaml

```diff
@@ -1,64 +1,30 @@
-# GnomAD VCF sites: 
+# GnomAD VCF sites:
 # - https://gnomad.broadinstitute.org/downloads
 # - https://macarthurlab.org/2018/10/17/gnomad-v2-1/
 # - http://ftp.ensemblorg.ebi.ac.uk/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/r2.1/genomes/
-# Script 
-# - sorts according to the reference file
-# - no need in decomposing (it is done in gnomad genome 2.1), but normalization and uniq are needed
-# - filters only PASS variants (segdup, decoy are retained)
-# - removes many INFO fields to reduce file size: controls_, hist, non_, using bcftools annotate instead of vt rminfo, because it failed randomly on chr1
-# - merges all chr into a single file
-# wget is separated from the processing because wget in the pipe fails randomly (on some systems?): it comes out when downloading many files.
 ---
 attributes:
   name: gnomad
-  version: 2.1
+  version: 2.1.1
 recipe:
   full:
     recipe_type: bash
     recipe_cmds:
       - |
-        url_prefix=http://ftp.ensemblorg.ebi.ac.uk/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/r2.1/genomes/
-        vcf_prefix=gnomad.genomes.r2.1.sites.chr
-        
-        ref=../seq/GRCh37.fa
+        url_prefix=https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/exomes/
```

Comment:
@roryk It is better to change url_prefix to vcf_prefix as in the other gnomad files